src/main.c

```diff
@@ -613,8 +613,12 @@ int main(int argc, char **argv)
 		}
 	// If username but no password given, interactively ask user
 	if (!cfg.password_set && cfg.username[0] != '\0') {
-		read_password(cfg.pinentry,
-		              "password", "VPN account password: ",
+		char hint[USERNAME_SIZE + 1 + REALM_SIZE + 1 + GATEWAY_HOST_SIZE + 12];
```

Comment:
should be 10 here if I'm not mistaken

Reply:
Indeed, `"_password\0"` is 1 + 8 + 1 = 10 characters long.